app/components/bali/tags/component.rb

```diff
@@ -0,0 +1,44 @@
+# frozen_string_literal: true
+
+module Bali
+  module Tags
+    class Component < ApplicationViewComponent
+      renders_many :tag_items, ->(text: nil, href: nil, delete: false, **options) do
+        @withlinks = true if delete && text.to_s.length.positive? && href.present?
+        size = @all_sizes || nil
+        Bali::Tag::Component.new(
+          text: text,
+          href: href,
+          delete: delete,
+          light: @light,
+          rounded: @rounded,
+          size: size,
+          **options
+        )
+      end
+
+      def initialize(
+        sizes: nil,
+        light: false,
+        rounded: false,
+        **options
+      )
+        @withlinks = false
+        @light = light
+        @rounded = rounded
+        @all_sizes = sizes
+        @options = prepend_class_name(options, 'tags-component')
+        @options = prepend_class_name(@options, "are-#{sizes}") if sizes.present?
+      end
+
+      def render?
+        tag_items.size.positive?
+      end
+
+      def container_options
+        @options = prepend_class_name(@options,
+                                      @withlinks ? 'field is-grouped is-grouped-multiline' : 'tags')
```

Comment:
Se ve mal si siempre le pones el field.is-gropued.is-multiline aunque no tengan links?

```suggestion
        @options = prepend_class_name(@options, 'field is-grouped is-grouped-multiline')
```

Reply:
No son clases necesarias, pero falta la otra opción y la clase "tags" sí que es necesaria cuando no trae links porque esa clase agrupa los .tag